.github/workflows/latestRelease.yml

```diff
@@ -29,7 +35,12 @@ jobs:
         name: Update Sonar
         run: |
           cd webapp_frontend
+          echo "${{ secrets.SSL_CERT }}" > filefighter.de_ssl_certificate.cer
+          yes yes | keytool -import -alias filefighterde -file filefighter.de_ssl_certificate.cer  -storetype JKS -keystore $JAVA_HOME/lib/security/cacerts  -storepass changeit
```

Comment:
i mean what is `yes yes` supposed to mean lol

Reply:
yes yes

Reply:
https://www.computerhope.com/unix/yes.htm